### DISCUSSION TITLE
Making UI server always up and running without manually any command or docker

### DISCUSSION THREAD

**Discussion by thexmeta** [ArchiveBox/ArchiveBox#1257] [Q&A]
HI, i want to run the archivebox server in windows without docker and by manually running "archivebox server 0.0.0.0:8000 " each time i wanted to access. I want to host server always up and running.

**Comment by pirate** [comment]
This is not supported, I can only provide support for Docker on Windows. Some users have gotten it working in WSL 2 or even outside the WSL but I don't own a Windows system and don't have the bandwidth to officially support Windows without Docker.
